app/lib/shop.rb

```diff
@@ -0,0 +1,29 @@
+require 'json'
+
+class Shop
+  attr_reader :menu, :all_shop_information, :shop_file
+
+  def initialize
+    @shop_file = 'hipstercoffee.json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.